ui/components/component-library/button-base/README.mdx

```diff
@@ -98,6 +98,22 @@ import { ButtonBase } from '../../component-library';
 <ButtonBase href="/metamask">Anchor Element</ButtonBase>;
 ```
 
+### External link
+
+When an `externalLink` prop is passed it will change the element to an anchor(`a`) tag and add the `target="_blank"` and `rel="noopener noreferrer"` attributes.
+
+<Canvas>
+  <Story id="components-componentlibrary-buttonlink--external-link" />
+</Canvas>
```

Comment:
Apologies I think I made a mistake with the component id that renders the story in the Docs tab

```suggestion
  <Story id="components-componentlibrary-buttonbase--external-link" />
```

Will fix this

<img width="1440" alt="Screenshot 2023-03-09 at 12 23 21 PM" src="https://user-images.githubusercontent.com/8112138/224147336-39df00f9-6367-4123-a2f2-bcd637b9e2e2.png">